Header header
bool obstacle_found
geometry_msgs/Point location